src/plugins/paginate/paginate.js

```diff
@@ -249,7 +290,10 @@ $document.on( "click", "." + pagerClass + " a", function()  {
 
 		updateItems( elm );
 		goToPage( elm );
+		elm.scrollIntoView( { behavior: "smooth" }, true );
```

Comment:
I suggest adding a condition for the item to be out of view before scrolling to prevent useless scrolling.
```suggestion
		if ( elm.getBoundingClientRect().top < 0 ) {
			elm.scrollIntoView( { behavior: "smooth" }, true );
		}
```